docs/BestPractices.md

```diff
@@ -204,3 +205,42 @@ FROM node:alpine as app
 ## Copy built node modules and binaries without including the toolchain
 COPY --from=builder node_modules .
 ```
+
+
+## Smaller images without npm/yarn
+
+If you want to achieve an even smaller image size than the `-alpine`, you can omit the npm/yarn like this:
+
+```Dockerfile
+FROM node:18-alpine3.16 AS builder
+WORKDIR /build-stage
+COPY package*.json ./
+RUN npm ci
+
+# Copy the the files you need
+COPY . ./
+
+RUN npm run build
+
+
+# Make sure the alpine version is the same as in the build stage
+FROM alpine:3.16
+RUN apk add --no-cache libstdc++
+RUN apk add --no-cache dumb-init
```

Comment:
These two line can be merged:
`RUN apk add --no-cache libstdc++ dumb-init`

Reply:
Good catch! Thank you for pointing this out.